cmd/singularity/actions_test.go

```diff
@@ -176,6 +227,8 @@ func testSTDINPipe(t *testing.T) {
 		{"sh", "library", []string{"-c", "echo true | singularity shell library://busybox"}, 0},
 		{"sh", "docker", []string{"-c", "echo true | singularity shell docker://busybox"}, 0},
 		{"sh", "shub", []string{"-c", "echo true | singularity shell shub://singularityhub/busybox"}, 0},
+		// Test a Bad target pwd
```

Comment:
This ... is testing a good target for `--pwd` ... `/etc` exists in the container